models/User.php

```diff
@@ -225,6 +234,14 @@ public function beforeValidate()
         ) {
             $this->username = $this->email;
         }
+
+
+        /*
+         * Apply Password Length Settings
+         */
+        $minPasswordLength = self::getMinPasswordLength();
```

Comment:
Can you change `self` to `static` so that people can override the `getMinPasswordLength()` method if they want to?